components/latex_block/latex_block.test.tsx

```diff
@@ -12,28 +12,34 @@ describe('components/LatexBlock', () => {
         enableLatex: true,
     };
 
-    test('should match snapshot', () => {
+    test('should match snapshot', async () => {
         const wrapper = shallow(<LatexBlock {...defaultProps}/>);
+        await import('katex'); //manually import katex
```

Comment:
You might be able to change this to
```javascript
await Promise.resolve();
```
I think we just need to await on something to let the `LatexBlock`'s `import(...).then` be run, so it doesn't matter what exactly we're awaiting on here.